Makefile

```diff
@@ -65,3 +65,105 @@ build-image:
 .PHONY: push-image
 push-image:
 	docker push $(IMAGE)
+
+.PHONY: clean
+clean:
+	rm -rf ./build
+
+.PHONY: sdk-build
+sdk-build:
+	npm ci
+	npm run build
+
+.PHONY: react-native-npm-ci
+react-native-npm-ci:
+	cd ./example/reactnative; \
+		rm -rf node_modules; \
+		yarn install --frozen-lockfile
+
+.PHONY: react-native-set-versionCode
+react-native-set-versionCode:
+	/usr/bin/sed -I "" "s/versionCode 1/versionCode $(shell date +%s)/" ./example/reactnative/android/app/build.gradle
```

Comment:
What is this for? Seems not referenced anywhere. And I believe there should be other way to set the version code without using text replacement

Reply:
We can either:
1. Use environment variable: https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables
2. Use gradle properties: https://docs.gradle.org/current/userguide/build_environment.html#sec:project_properties

Reply:
Will improve this when I do the android part.